third_party/blink/renderer/core/dom/node.cc

```diff
@@ -329,6 +329,10 @@ Node::Node(TreeScope* tree_scope, ConstructionType type)
   // Document is required for probe sink.
   if (tree_scope_)
     probe::NodeCreated(this);
+
+  if (recordreplay::AreEventsDisallowed()) {
```

Comment:
This can definitely happen while we're diverged from the recording, client evals etc. could create new nodes.  So this could have a `&& !HasDivergedFromRecording()` or would the situation still be problematic if it happened?

Reply:
Yeah, I thought we had a fix for `HasDivergedFromRecording` somewhere already, but at least `NewIdAnyThread` still returns 0 even if `HasDivergedFromRecording`.

I'm thinking, I'll ship this as-is, and we can then make adjustments to `NewIdAnyThread` based on the information we get back from this.